### PR TITLE
feat(dailyquestions): Add index on the header

### PR DESCRIPTION
**Pull request by semia**
header_dailyquestion
Related to issue #118

### CODE REVIEW

File: app/views/layouts/_header.html.haml [hunk 1]
```diff
@@ -7,6 +7,7 @@
   - if user_signed_in?
     %li{class: active_class(my_profile_path), role: 'presentation'}= link_to "#{current_user.first_name} #{current_user.last_name}", my_profile_path
     %li{class: active_class(questions_path), role: 'presentation'}= link_to 'Questions', questions_path
+    %li{class: active_class(admin_daily_questions_path), role: 'presentation'}= link_to 'Liste quotidienne des questions', admin_daily_questions_path
```
Comment (arnlen) [inline app/views/layouts/_header.html.haml]: Maybe you could wait for the #135 to be merged, and then also add i18n for this partial?

**Comment by arnlen** [comment]
IMHO: the correct terme here would be: "Questions journalières".

**Comment by semia** [comment]
Oki Thx :-)

**Comment by arnlen** [comment]
Good to merge, thx! 👍

**Pull request merged**